src/rules/order.js

```diff
@@ -310,31 +306,32 @@ function computePathRank(ranks, pathGroups, path, maxPosition) {
   }
 }
 
-function computeRank(context, node, ranks, name, type, excludedImportTypes) {
+function computeRank(context, ranks, importEntry, excludedImportTypes) {
   let impType
-  if (type === 'import:object') {
+  let rank
+  if (importEntry.type === 'import:object') {
     impType = 'object'
   } else {
-    impType = importType(name, context)
+    impType = importType(importEntry.value, context)
   }
-  let rank
+  importEntry.group = impType
   if (!excludedImportTypes.has(impType)) {
-    rank = computePathRank(ranks.groups, ranks.pathGroups, name, ranks.maxPosition)
+    rank = computePathRank(ranks.groups, ranks.pathGroups, importEntry.value, ranks.maxPosition)
```

Comment:
@ljharb shal we rather pass `importEntry.displayName` to this function and allow to add path-groups for object-imports as well or leave it as it is (`importEntry.value`)?

I'd rather go for disallowing path-groups for object-imports by using `importEntry.value`.

Reply:
this seems fine as is